src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -338,6 +346,43 @@ public function callableMethodCommand(InputInterface $input, OutputInterface $ou
     {
         $output->writeln('from the code...');
     }
+
+    public function testSetProcessTitle()
+    {
+        $command = new TestCommand();
+
+        $this->assertSame('', cli_get_process_title());
+
+        $command->setProcessTitle('process title');
+        $inputMock = $this->prophesize(InputInterface::class);
+        $outputMock = $this->prophesize(OutputInterface::class);
+        $command->run($inputMock->reveal(), $outputMock->reveal());
+
+        $this->assertSame('process title', cli_get_process_title());
+    }
+
+    public function testSetAliases()
+    {
+        $command = new TestCommand();
+        $command->setAliases([]);
+        $command->setAliases(new TraversableDummy());
```

Comment:
This will already reveal a bug in the Command class btw, as `getAliases()` is documented to always return an array, but it returns a traversable when a traversable was passed. Solution is to use `iterator_to_array` in `setAliases()` when a traversable was passed or to change the PHPdoc that a traversable might be returned from `getAliases()`.